clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -2562,13 +2562,16 @@ raw2trace_t::write(raw2trace_thread_data_t *tdata, const trace_entry_t *start,
                 if (it->size == TRACE_MARKER_TYPE_TIMESTAMP)
                     tdata->last_timestamp_ = it->addr;
                 else if (it->size == TRACE_MARKER_TYPE_CPU_ID) {
+                    DR_CHECK(tdata->chunk_count_ > 0, "chunk_count_ should be +1");
```

Comment:
s/+1/+ve/?

Reply:
Reworded to "chunk_count_ should have been incremented already"